operator/controllers/csiscaleoperator_controller.go

```diff
@@ -164,12 +164,12 @@ func (r *CSIScaleOperatorReconciler) Reconcile(ctx context.Context, req ctrl.Req
 		logger.V(1).Info("Updated resource status.", "Status", cr.Status)
 	}()
 
-	r.Scheme.Default(instanceUnwrap)
-	err = r.Client.Update(context.TODO(), instanceUnwrap)
-	if err != nil {
-		logger.Error(err, "Reconciler Client.Update() failed")
-		return ctrl.Result{}, err
-	}
+	// r.Scheme.Default(instanceUnwrap)
```

Comment:
is this being relied on for "upgrade" purposes from previously created CSIScaleOperator resources?  If so, this should not be done via mutating existing spec IMO.  It is better to bump CSIScaleOperator version and use conversion webhooks.

Reply:
default should already be handled by Schema as is for newly created resources